src/features/island/hud/components/codex/pages/FactionLeaderboard.tsx

```diff
@@ -305,7 +306,7 @@ export const FactionDetails: React.FC<{
 
       <div className="flex justify-between font-secondary text-xs pt-1">
         <span>
-          {t("last.updated")} {getRelativeTime(leaderboard.lastUpdated)}
+          {t("last.updated")} {getRelativeTime(leaderboard.lastUpdated, now)}
```

Comment:
This could use the same component I mentioned above. That would keep it live which it really should be.